.github/workflows/nightly_release.yaml

```diff
@@ -0,0 +1,28 @@
+name: Nightly Release
+
+on:
+  workflow_call:
+
+jobs:
+    releaseDate:   
+        runs-on: ubuntu-latest
+        outputs:
+            date: ${{steps.releaseDate.outputs.date}}
+        steps:
+            - name: output current date
+              id: releaseDate
+              run: echo "date=$(date +'%Y-%m-%d')" >> $GITHUB_OUTPUT
+    createBranch: 
```

Comment:
instead of creating a branch every time it would be good to always use the same branch. something called releases/nightly or something.